pkg/platform/authentication/auth.go

```diff
@@ -196,6 +197,9 @@ func (s *Auth) AuthenticateWithModel(credentials *mono_models.Credentials) error
 			if os.IsTimeout(err) {
 				return locale.NewInputError("err_api_auth_timeout", "Timed out waiting for authentication response. Please try again.")
 			}
+			if api.ErrorCode(err) == 403 {
+				return locale.NewInputError("err_auth_forbidden", "You are not allowed to login now. Please try again later.")
+			}
```

Comment:
nit: Do we want to update this error message to provide more context to the user? If this isn't in the swagger spec then the mono API isn't returning it so it could be something to do with the user's network configuration. We also may want them to try to report it to the forums.